### PR TITLE
feat(common): create from _helpers

### LINKED ISSUE #169: packaging: make _helpers a regular package
**Current behavior**

we are putting common things to `_helpers` which get's imported across packages and produces duplicate ts declaration files and makes things not DRY

**Expected behavior**

We should rename it to `core` or `common` or `base` ( let's discuss it before proceeding ) and make it regular package, so when used across other packages, only one one is used instead of bundling it to every package which uses it

### PR DESCRIPTION
**Pull request by devopsmariocom**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/wc-catalogue/blaze-elements/blob/master/docs/CONTRIBUTING.md#commit
- [x] There are no linting errors and code is properly formatted (your run before push `yarn ts:format && yarn ts:lint:fix`)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

Closes #169 
- _helpers is simple folder
- packages are using relative paths for other packages

**What is the new behavior?**
- _helpers are now common package
- packages are linked using proper dependecies 

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by Hotell** [comment]
readability nitpick, the title is "incomplete"

consider `feat(common): create regular package from _helpers`.

Also pls always provide links to existing issue, instead of writting by hand **What is the current behavior** section

**Comment by devopsmariocom** [comment]
@Hotell tests are failing for unknown reason on CI, please meanwhile try go thru my changes.

Thanks

File: packages/blaze-elements/index.demo.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import '@blaze-elements/button/index.demo';
 import '@blaze-elements/calendar/index.demo';
 import '@blaze-elements/card/index.demo';
 import '@blaze-elements/collapsible/index.demo';
+import '@blaze-elements/common/index.demo';
```
Comment (Hotell) [inline packages/blaze-elements/index.demo.tsx:11]: aha!

**Review by Hotell — COMMENTED** [review]
pls do not merge. Tests are failing and there are some TS errors after our paired Code Review

**Comment by Hotell** [comment]
depends on https://github.com/wc-catalogue/blaze-elements/issues/230

**Review by Hotell — CHANGES_REQUESTED** [review]
depends on https://github.com/wc-catalogue/blaze-elements/issues/230. then merge with https://github.com/wc-catalogue/blaze-elements/pull/229

**Review by Hotell — APPROVED** [review]

**Pull request merged**